ios/build.sh

```diff
@@ -135,6 +136,11 @@ echo "Building KMEI..."
 rm -r $KMEI_BUILD_PATH/$CONFIG-universal 2>/dev/null
 xcodebuild -quiet -project engine/KMEI/KeymanEngine.xcodeproj -target KME-universal -configuration $CONFIG \
   $CODE_SIGN_IDENTITY $CODE_SIGNING_REQUIRED $DEV_TEAM
+
+if [ $? -ne 0 ]; then
+  fail "KMEI build failed."
+fi
+
 assertDirExists "$BUILD_FRAMEWORK_PATH"
```

Comment:
Now that ll.152-153 are commented out, maybe the `assertDirExists` check in l.144 can move inside the 
`if [ $BUILD_NUMBER ]; then` block.

Same for l.193